### ISSUE TITLE
The icon in Brave browser shows just a question mark not a count.

### ISSUE THREAD

**Issue by tomhusband** [Athou/commafeed-browser-extension#3]
I installed in Brave on my Linux Mint laptop. I've linked it to my PikaPod instance and can open CommaFeed in a new tab but the count in the Brave icon remains as a question mark. How do I get the count to show?

**Comment by Athou** [comment]
You need to open the extension options page and enter the URL to your CommaFeed instance. The default URL points to commafeed.com and since you're not logged in there the extension shows a `?`.

The options page should have been opened automatically when the extension was installed though. Was that not the case?

**Comment by tomhusband** [comment]
Yes, the option page is pointed to the PikaPod instance. The same link I use to open it in the browser. When I click the extension CommaFeed opens correctly. It just doesn't show the number of feeds.

**Comment by Athou** [comment]
I don't know then, I just installed Brave and the extension from the Chrome marketplace and it works fine.

You could try to go to brave://extensions/?id=bpbfpjiciblcfeganojjkfapnllbhdga, enable Developer Mode in the top right corner, click on Inspect views -> service worker, go to the network tab, wait a minute until the extensions tries to refresh the count and see what the error is.

**Comment by tomhusband** [comment]
Strange, I don't see any errors. I'll live with it. It's still handy for a quick way to open the app.
